rs/nervous_system/canisters/src/registry.rs

```diff
@@ -171,6 +230,26 @@ impl FakeRegistry {
                 .store(version, std::sync::atomic::Ordering::SeqCst);
         }
     }
+
+    /// Applies mutations as latest version.  This is similar to what Registry actually does, which
+    /// is helpful for re-using other code that sets initial Registry mutations.
+    pub fn apply_mutations(&self, mutations: Vec<RegistryMutation>) {
+        let current_version = self.version.load(std::sync::atomic::Ordering::SeqCst);
+        let next_version = current_version + 1;
```

Comment:
Should we use `checked_add` here?

Reply:
I don't think so, b/c it's a test function, but changed it anyway